source/02-combat-prototype/actors/physics/Jump.gd

```diff
@@ -1,4 +1,4 @@
-extends PlatformPhysics
+extends "res://actors/physics/PlatformCommand.gd"
 
 signal available_jumps_depleted
```

Comment:
Maybe `depleted` and `replenished` are enough, I'm not sure with how explicit this information should be, since it communicates a very specific state of this object....